posts/2022/02/what-do-we-need-to-write.md

```diff
@@ -0,0 +1,126 @@
+<!--
+.. title: What do we need to write?
+.. slug: what-do-we-need-to-write
+.. date: 2022-02-22 14:00:00 UTC+01:00
+.. author: Tania Allard, Kate Brack, Tony Fast, Rohit Goswami, Eric Kelly, Mars Lee, Dharhas Pothina, Isabela Presedo-Floyd, Dillon Roach, Melissa Weber Mendonça
+.. tags: Writing
+.. category:
+.. link:
+.. description:
+.. type: text
+.. previewimage:
+-->
+
+We write code. We write issues. We write documentation. We write notes to 
+ourselves, messages to each other, and guidelines to unite teams across 
+projects.
+
+Day in and out our remote work and open source lives are driven by written 
+communication. But blog posts are one kind of writing that eludes our regular 
+practice. In our weekly show and tell we got real about "why can writing blog 
+posts be so hard?" and collaboratively wrote up this blog post about what we learned 
+from the discussion.
+
+<!-- TEASER_END -->
+
+A quick note about language: At Quansight, we have many people who are 
+multilingual and/or are not working in their first language. This is the norm 
+here. Because of this, we know that there are language barriers to writing that 
+were only mentioned implicitly in the discussion, but have to be overcome on a daily
+basis by the team.
+
+## What obstacles prevent us from publishing writing?
+
+> Writing is not a singular task. There's getting the idea, organizing it, 
+editing what you have, getting feedback, publishing, and so on…
+
+We found ourselves split into two camps (1) those that struggle to start 
+writing and (2) those that struggle to complete and publish their writing. 
+Below are their stories:
```

Comment:
```suggestion
Below are their stories.
```

I think a colon is not necessary here because there's no list of items right below it.